public/manifest/production.json

```diff
@@ -18,5 +18,44 @@
       "display_name": "Ethiopia",
       "isd_code": "251"
     }
+  ],
+  "v2": [
+    {
+      "country_code": "IN",
+      "display_name": "India",
+      "isd_code": "91",
+      "deployments": [
+        {
```

Comment:
I envision a world where each deployment can start overriding things from its parent, like ISD code - that's not a great example, but the flexibility is great.